tests/WeatherParserUtilsTests.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const WeatherParserUtils = require('../lib/Position/WeatherParserUtils.js');
+
+const chai = require('chai');
+const expect = chai.expect;
+
+describe('WeatherParserUtils', () => {
+    it('Text regex & extract comment from weather', () => {
+        const parsed = WeatherParserUtils.parseWeatherData('COMMENT_BEFORE_g002t017r000p000P000h80b10232L016_COMMENT_AFTER');
```

Comment:
Is it real scenario? In "APRS protocol reference" comment is always at the end

Reply:
> I would really opt for not using regex here. If you don't have time to change it I can create PR tomorrow.


Yes, I'd love to see what you come up with.  I figured we'd have to collaborate on this a bit.